detection-rules/attachment_malicious_adobe_branding.yml

```diff
@@ -0,0 +1,40 @@
+name: "Attachment: Malicious Adobe-branded file"
+description: |
+  The attached file uses Adobe branding to download another file, as employed by threat actors, such as Qakbot.
+type: "rule"
+references:
+  - "https://bazaar.abuse.ch/sample/81fa36b1d5e9f5457644fc784f2e03d14734052c35023e899ba521697d0bbef9"
+  - "https://bazaar.abuse.ch/sample/a870526ed1b0c71d20eb6aa3549566622ea563eece594fe018d018d9977f3afa"
+  - "https://twitter.com/unit42_intel/status/1602715058383065091"
+  - "https://www.fortinet.com/blog/threat-research/delivery-of-malware-phishing-campaigns-in-q3-2022"
+  - "https://twitter.com/pr0xylife/status/1602604206346608641"
+severity: "high"
+source: |
+  type.inbound 
+       and any(attachments,
+           (
+              .file_extension in~ ("html", "htm", "pdf") or
+              .file_extension in~ $file_extensions_common_archives or
+              .file_type == "html"
```

Comment:
have we seen this delivered via HTML files, or strictly PDF files?

Reply:
I've seen both in testing.